### PR TITLE
examples/bluetooth: Supplement bt_unit_test code

### PR DESCRIPTION
**Pull request by Wootak**
framework/bluetooth: Add set_local_name function,

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [12441f1b05ae1f12424786efc49bf2fd21727af8] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [12441f1b05ae1f12424786efc49bf2fd21727af8] - Code Rule Check OK.

File: apps/examples/bluetooth/bt_unit_test.c [hunk 1]
```diff
@@ -265,6 +256,32 @@ static void __le_scan_result_cb(int result, bt_adapter_le_device_scan_result_inf
...
+	int ret;
+	char *name = NULL;
+
+	TC_PRT("__state_changed_cb\n");
+	TC_PRT("result: %s\n", __bt_get_error_message(result));
+	TC_PRT("state: %s\n", (adapter_state == BT_ADAPTER_ENABLED) ? "ENABLED" : "DISABLED");
+
+//	adapter_enabled = ((adapter_state == BT_ADAPTER_ENABLED) ? true : false);
+
+	ret = bt_adapter_get_name(&name);
```
Comment (RoyAnupam) [inline apps/examples/bluetooth/bt_unit_test.c]: Can we check adapter is enabled or not before calling remaining functions?

**Comment by seinfra** [comment]
Target : [14b967484380787200bae25db2af8386c727ce11] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [14b967484380787200bae25db2af8386c727ce11] - Code Rule Check (C++) OK.

File: apps/examples/bluetooth/bt_unit_test.c (hunk 1 above)
Comment (Wootak) [reply apps/examples/bluetooth/bt_unit_test.c]: Done. Thank you.

**Pull request merged**